cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -173,92 +197,21 @@ EXECUTE PROCEDURE update_last_update_timestamp();
 --- Create pre-computed tables.
 --- These are required to ensure acceptable latency for the tileserver.
 
--- Demographic aggregation tables.
-
-CREATE TABLE IF NOT EXISTS state_demographics
-(
-    census_geo_id         varchar(255) NOT NULL,
-    name                  varchar(255) NOT NULL,
-    black_population      real,
-    white_population      real,
-    total_population      real,
-    under_five_population real,
-    poverty_population    real,
-    geom                  geometry(Geometry, 3857),
-    PRIMARY KEY (census_geo_id)
-);
-CREATE INDEX IF NOT EXISTS geom_index ON state_demographics USING GIST (geom);
-
-CREATE TABLE IF NOT EXISTS county_demographics
-(
-    census_geo_id         varchar(255) NOT NULL,
-    name                  varchar(255) NOT NULL,
-    black_population      real,
-    white_population      real,
-    total_population      real,
-    under_five_population real,
-    poverty_population    real,
-    geom                  geometry(Geometry, 3857),
-    PRIMARY KEY (census_geo_id)
-);
-CREATE INDEX IF NOT EXISTS geom_index ON county_demographics USING GIST (geom);
-
--- TODO(breuch): Add insert statement once the zipcode -> demographics
--- connection established.
-CREATE TABLE IF NOT EXISTS zipcode_demographics
-(
-    census_geo_id         varchar(255) NOT NULL,
-    zipcode               varchar(255) NOT NULL,
-    black_population      real,
-    white_population      real,
-    total_population      real,
-    under_five_population real,
-    poverty_population    real,
-    geom                  geometry(Geometry, 3857),
-    PRIMARY KEY (census_geo_id)
-);
-CREATE INDEX IF NOT EXISTS geom_index ON zipcode_demographics USING GIST (geom);
-
 -- Lead connections aggregation tables.
 
 CREATE TABLE IF NOT EXISTS state_lead_connections
```

Comment:
Is there a reason this isn't in the `states` table? That would make it behave more like other tables like `parcels`.

Reply:
Good point - I didn't create this table in this PR, it was created when I created aggregate tables for lead connections (see deleted tables for counties + zips below). At that point it was because we generally were separating out the aggregated data into different tables that were only being used for function sources, and state / county / zip tables only stored information directly from the cartographic boundaries source. Agreed that it makes sense to unify these metrics into the state table now that we aren't storing by source.

Reply:
As discussed offline this will require a good amount of refactoring to pull from multiple sources. Left a TODO and will document as p1 on eng tasks if it isn't tracked yet.